### ISSUE TITLE
a-tags not accessible by tabbing, thus not accepted in accessibility tests

### ISSUE THREAD

**Issue by roggaj** [cornflourblue/jw-vue-pagination#23]
add role="button" or href="#" and prevent default bebahivor to pass these accessibility tests.

`<li v-for="page in pager.pages" :key="page" class="page-item page-number" :class="{'active': pager.currentPage === page}" :style="liStyles">
            <a class="page-link" @click="setPage(page)" :style="aStyles">{{page}}</a>
        </li>`

Need to be changed to 

```
<li v-for="page in pager.pages" :key="page" class="page-item page-number" :class="{'active': pager.currentPage === page}" :style="liStyles">
            <a class="page-link" href="#" @click.prevent="setPage(page)" :style="aStyles">{{page}}</a>
        </li>
```
